typesystem/namespaces.py

```diff
@@ -1,12 +1,12 @@
 import typing
 from collections.abc import MutableMapping
 
-from typesystem.schemas import Schema
+from typesystem.fields import Array, Field, Nested, Object
 
 
 class SchemaNamespace(MutableMapping):
     def __init__(self) -> None:
-        self._namespace = {}  # type: typing.Dict[str, typing.Type[Schema]]
+        self._namespace = {}  # type: dict
```

Comment:
is there a reason to keep using comments instead of real annotation syntax?

Reply:
Don’t recall what versions that starts being supported in?

Reply:
Oh right, 3.6. Yeah could defo consider a PR changing that across the board.

Reply:
see #50